posts/2024/2024_06_07_Inigo_week_1.rst

```diff
@@ -0,0 +1,29 @@
+First Week into GSoC 2024: Building the AutoEncoder, preliminary results
+========================================================================
+
+.. post:: June 7 2024
+    :author: Iñigo Tellaetxe
+    :tags: google
+    :category: gsoc
+
+
+What I did this week
```

Comment:
Hi @itellaetxe!

Nice blog, clear and concise, Great job.

Reply:
Thanks Wachiou!